### PR TITLE
Do not check for JIRA marker in github generated commit message

### PR DESCRIPTION
**Pull request by siong-chin**

### CODE REVIEW

**Review by rockdreamer — COMMENTED** [review]
Could you add a couple of tests for this please?

**Comment by siong-chin** [comment]
Thanks for the reminder. Tests added.

File: main/githooks.py [hunk 1]
```diff
@@ -806,6 +809,20 @@ def _test(input, is_jira=True):
...
 
 
+class TestCheckCommitMessage(unittest.TestCase):
+    def test_various_strings(self):
+        def _test(input, is_good=True):
+            rc = check_commit_msg(input, [])
+            self.assertEqual(rc == 0, is_good)
+        _test('ABC-1234')
+        _test('Some changes for ABC-1234 ticket')
+        _test('Trivail change NO_JIRA')
```
Comment (MatthewT123) [inline main/githooks.py]: spelling - Trivial not Trivail

**Review by MatthewT123 — APPROVED** [review]

File: main/githooks.py (hunk 1 above)
Comment (siong-chin) [reply main/githooks.py]: Fixed

**Pull request merged**